analyzers/tests/SonarAnalyzer.Test/Helpers/KnownAssemblyTest.cs

```diff
@@ -255,103 +255,115 @@ public void Combinator_StartsWith_Start_Or_EndsWith_End_2(string name, bool expe
     }
 
     [TestMethod]
-    public void XUnitAssert_2_4()
-    {
-        var compilation = TestHelper.CompileCS("// Empty file", NuGetMetadataReference.XunitFramework("2.4.2").ToArray()).Model.Compilation;
-        compilation.References(XUnit_Assert).Should().BeTrue();
-    }
+    public void XUnitAssert_2_4() =>
+        CompilationShouldReference(NuGetMetadataReference.XunitFramework("2.4.2"), XUnit_Assert);
 
     [TestMethod]
-    public void XUnitAssert_1_9()
-    {
-        var compilation = TestHelper.CompileCS("// Empty file", NuGetMetadataReference.XunitFrameworkV1.ToArray()).Model.Compilation;
-        compilation.References(XUnit_Assert).Should().BeTrue();
-    }
+    public void XUnitAssert_1_9() =>
+        CompilationShouldReference(NuGetMetadataReference.XunitFrameworkV1.ToArray(), XUnit_Assert);
```

Comment:
`CompilationShouldReference` accepts an IEnumerable
```suggestion
        CompilationShouldReference(NuGetMetadataReference.XunitFrameworkV1, XUnit_Assert);
```

Reply:
Same for the others.